t/plugin/kafka-logger.t

```diff
@@ -1135,7 +1135,7 @@ qr/failed to send data to Kafka topic: .*, brokers: \{"127.0.0.127":9092}/
                                 "key" : "key1",
                                 "timeout" : 1,
                                 "include_req_body": true,
-                                "request_body_expr": [
+                                "include_req_body_expr": [
                                     [
                                       "remote_addr",
```

Comment:
We can do it with `arg_xx`. There is no need to use a separate rule.

Reply:
I'm not understand what this means

Reply:
Use `arg_blah == 1` in the expression, and hit it with "POST /hello?blah=1" and "POST /hello?blah=2".

Reply:
> Use `arg_blah == 1` in the expression, and hit it with "POST /hello?blah=1" and "POST /hello?blah=2".

I'm using `ctx.var` to eval the expr and  `ctx.var` don't have request param, it only has request urI.If you want to achieve this, i need to eval the expr by `ngx.req.get_uri_args`

Reply:
I'm using ctx.var to eval the expr and ctx.var doesn't directly hold args. it only has args table. If you want to achieve this, i need to eval the expr by ctx.var.args

Reply:
I mean to use `$arg_` variable. You can refer to https://github.com/apache/apisix/blob/cc43b9fc1bf9be14c05d36415f83cdd189d0a7f5/t/plugin/traffic-split4.t#L110

Reply:
> I mean to use `$arg_` variable. You can refer to
> 
> https://github.com/apache/apisix/blob/cc43b9fc1bf9be14c05d36415f83cdd189d0a7f5/t/plugin/traffic-split4.t#L110

done ,thank you very much